### PR TITLE
Edited Display of Price in Price Details

### PR DESCRIPTION
**Pull request by akblal**
Updates:
- Put strikethrough original price with sale price in red font, ONLY IF there is a sale on the product

Please review: @Louisaflor @noattongi @xiaxia330

### CODE REVIEW

**Review by Louisaflor — APPROVED** [review]
looks good!

File: client/src/Overview/StyleSelector/StyleSelectorOverview.jsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ function StyleSelectorOverview ({setSelectedStyle, setOriginalPrice, setSalePric
     .then(res => {
       let styleGuide = res.data;
       setStyles(styleGuide.results)
+      console.log (styleGuide.results, 'this is the styleguide')
```
Comment (noattongi) [inline client/src/Overview/StyleSelector/StyleSelectorOverview.jsx]: don't forget to remove console logs for PRs

Comment (akblal) [reply client/src/Overview/StyleSelector/StyleSelectorOverview.jsx]: roger that! will push another PR request as soon as I remove it.

**Review by noattongi — APPROVED** [review]
nice job!

**Pull request merged**